scripts/src/main/resources/scripts/command/java

```diff
@@ -55,6 +55,16 @@ function doMigrate() {
   exit ${?}
 }
 
+function doDemoSetup() {
+  local mtsj_branch="develop"
+  local mtsj_repo="my-thai-star-${mtsj_branch}"
+  local mtsj_dir="${DEVON_IDE_HOME}/workspaces/main/${mtsj_repo}/java/mtsj/"
+  doDownload "https://github.com/devonfw/my-thai-star/archive/${mtsj_branch}.zip"
```

Comment:
developers do `git clone` on projects. Downloading a zip is a bad idea. How would you later update the project if MTS has been updated?